server.js

```diff
@@ -24,10 +25,19 @@ app.use(
 app.use(compression());
 
 const gitClone = (repoUrl) => {
+  const parsedUrl = new URL(repoUrl);
+
+  const userInfo =
+    parsedUrl.username && parsedUrl.password
+      ? `${parsedUrl.username}:*****`
+      : parsedUrl.username || "";
+
+  const sanitizedRepoUrl = `${parsedUrl.protocol}//${userInfo}${parsedUrl.host}${parsedUrl.pathname}`;
+
   const tempDir = fs.mkdtempSync(
     path.join(os.tmpdir(), path.basename(repoUrl))
   );
-  console.log("Cloning", repoUrl, "to", tempDir);
+  console.log("Cloning", sanitizedRepoUrl, "to", tempDir);
```

Comment:
Let's remove the log completely since there could be stuff in query params etc?

Reply:
What if we just log the path then? i think it's still valuable to see which git repos are being cloned.

Reply:
Sounds good to me.